manifests/worker.pp

```diff
@@ -123,14 +140,12 @@
         'encrypt'          => $consul_encrypt,
         'ui'               => $consul_ui,
       },
-      services    => {
-        'mesos-slave' => {
-          port => 5051
-        },
-      },
       require     => Package['unzip'],
     }
   }
+  consul::service { 'mesos-slave':
+    port => 5051
+  }
```

Comment:
I'm guessing if we're a `$controller_worker` we'd already have consul set up for the controller and we'd just need the service. That being the case, should we still be passing `bootstrap_expect` to the non-controller consul (which has `server` set to `false`)?